sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/theme/Dimension.kt

```diff
@@ -1,8 +1,7 @@
+@file:Suppress("MagicNumber")
+
 package co.nimblehq.sample.compose.ui.theme
 
 import androidx.compose.ui.unit.dp
 
-@Suppress("MagicNumber")
-object Dimension {
-    val SpacingNormal = 16.dp
-}
+val SpacingNormal = 16.dp
```

Comment:
Shall we make it to be a compile-time constant by using `const val`?
since this dimension will never change. 🙏🏻

Reply:
@lydiasama I think it's not possible since the dimen variable has type DP, which is not a primitive type 🙏🏼

![Screen Shot 2565-08-22 at 11 33 54](https://user-images.githubusercontent.com/13492460/185840211-846c32b0-25ca-44e5-851e-af6eb41f57ab.png)

Reply:
Oh, it's true. 😂